examples/datadog/agent_on_openshift_values.yaml

```diff
@@ -0,0 +1,44 @@
+# Datadog Agent with Logs, APM, Processes, and System Probe enabled
+# with specific configurations to work on OpenShift 4.
+# When installing the chart, install onto a non-default namespace with
+# `helm install --namespace <your_ns>` due to existing SecurityContextConstraints on the default namespace.
+# To setup the SCC in the agent namespace, update "system:serviceaccount:default:datadog-agent"
+# with "system:serviceaccount:<your_ns>:<your_sa>" in
```

Comment:
There are other ways in OpenShift 4. We can add an RBAC to allow usage of a specific SCC.

Reply:
maybe instead of these comments, i will link to our docs which seem to provide a good level of detail around this